assets/src/components/conversations/ConversationsDashboard.tsx

```diff
@@ -285,9 +314,13 @@ class ConversationsDashboard extends React.Component<Props, State> {
   };
 
   render() {
-    const {selectedConversationId, closing = []} = this.state;
+    const {selectedConversationId, pagination = {}, closing = []} = this.state;
     const {title, conversationIds = []} = this.props;
     const loading = this.props.loading || this.state.loading;
+    const hasMoreConversations =
```

Comment:
Can check if `pagination.after` is not null, and use that to indicate if there's more conversations?

Reply:
i renamed `after`/`before` to `next`/`previous`, so we do the check for `pagination.next` below :+1:

(there's an edge case where we've already loaded all the conversations into the cache, which is why we also check that `conversationIds.length < pagination.total`)